Sources/BrowserServicesKit/PrivacyConfig/ExperimentCohortsManager.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  ExperimentCohortsManager.swift
```

Comment:
did you consider putting it in a separate module?

Reply:
I did but It seems pretty contained so I am not sure it is worth it